semanticdb/scalac/library/src/main/scala-2.13.0-/scala/meta/internal/semanticdb/scalac/SemanticdbReporter.scala

```diff
@@ -17,13 +17,11 @@ class SemanticdbReporter(underlying: Reporter) extends StoreReporter {
       case 2 => underlying.error(pos, msg)
       case _ =>
     }
-    // underlying reporter may have filtered out the message
-    updateCounts()
```

Comment:
Most likely when using zinc those counts were never updated:

https://github.com/sbt/zinc/blob/fc58a39402ca7abfda17133a5b3a43de3344ca49/internal/compiler-bridge/src/main/scala/xsbt/DelegatingReporter.scala

The only sensible way to get if errors exist is to use the hasErrors method